.betterer.ts

```diff
@@ -10,8 +10,28 @@ export default {
       .include('**/*.{ts,tsx}')
       .exclude(/public\/app\/angular/),
   'no undocumented stories': () => countUndocumentedStories().include('**/!(*.internal).story.tsx'),
+  'no borderRadius without token': () => countBorderRadiusWithoutToken().include('**/*.ts*'),
 };
 
+function countBorderRadiusWithoutToken() {
+  return new BettererFileTest(async (filePaths, fileTestResult) => {
+    await Promise.all(
+      filePaths.map(async (filePath) => {
+        // look for borderRadius in the story file and check if it uses the token
+        const regexBorder = new RegExp('border.*radius:.*', 'gmi');
+        const regexToken = new RegExp('theme.border.radius.*', 'gm');
```

Comment:
I think we would want to test if these two are on the same line no? If somewhere else in the file another border-radius uses the tokens it doesn't mean that the file "passes".
We should also check the results of that betterer file, I checked some of those files and can't really understand why some scenarios are there, like `SearchCard` or `Button`

Reply:
Ok, copy that. I will change them to one regex that checks that.
I think on `Button.tsx` is because they are using `className={cx(styles.button, className)}` and `className={linkButtonStyles}`, as in `SearchCard.tsx` you can find `className={styles.card}`, `className={styles.image}` and so on.

Reply:
but this one is about borderRadius no? 🤔

Reply:
Sorry, I mixed issues.
In `SearchCard` there is a `border-radius: ${theme.shape.radius.default};` where it says `shape` instead of `border`.
In `Button`is the same, `borderRadius: theme.shape.radius.default`, so I should have to change the regex to `theme.*.radius.*`, shouldn`t I? Or do we want to avoid using `theme.shape`?

Reply:
I didn't even notice that either, but the token is `theme.shape.radius.` not `theme.border.radius.` 😅  so just need to check for the shape one 👍

Reply:
I think you might be better off making this an eslint rule tbh

Reply:
+1 to this. think we should make an eslint rule and just fix the remaining cases. skim reading the code search results, i'd guess there's maybe 20 left, and they don't look super tricky to just switch to the tokens tbh.